Sources/BranchSDK/BNCServerRequestQueue.m

```diff
@@ -284,6 +284,19 @@ - (void)retrieve {
     }
 }
 
+// It's been reported that unarchive can fail in some situations. In that case, remove the queued requests file.
+- (void)removeSaveFile {
+    NSURL *fileURL = [BNCServerRequestQueue URLForQueueFile];
+    if (fileURL) {
+        NSError *error;
+        [NSFileManager.defaultManager removeItemAtURL:fileURL error:&error];
+        
+        if (error) {
+            [[BranchLogger shared] logError:@"Failed to remove archived queue" error:error];
```

Comment:
Can you add a test to capture the fail cases?

Reply:
Not easily for a couple reasons. The class itself needs to be redesigned for testability. Second this scenario is a disk write which cannot be run in parallel with other tests without potentially impacting them.